pkg/analytics/track.go

```diff
@@ -62,6 +62,7 @@ const (
 	disableEvent             = "Disable Analytics"
 	stackNotSupportedField   = "Stack Field Not Supported"
 	buildPullErrorEvent      = "BuildPullError"
+	deleteContexts           = "Contexts Deletion"
```

Comment:
```suggestion
	deleteContexts           = "Contexts Delete"
```

Reply:
@AdrianPedriza I would keep the name simpler, just "Delete" instead of "Deletion". WDYT?